descriptors/ModuleDescriptor-template.json

```diff
@@ -2,40 +2,9 @@
   "id": "${artifactId}-${version}",
   "name": "Orders CRUD module",
   "provides": [
-    {
-      "id": "orders-storage.adjustments",
-      "version": "3.0",
-      "handlers": [
-        {
-          "methods": ["GET"],
-          "pathPattern": "/orders-storage/adjustments",
-          "permissionsRequired": ["orders-storage.adjustments.collection.get"]
-        },
-        {
-          "methods": ["POST"],
-          "pathPattern": "/orders-storage/adjustments",
-          "permissionsRequired": ["orders-storage.adjustments.item.post"]
-        },
-        {
-          "methods": ["GET"],
-          "pathPattern": "/orders-storage/adjustments/{id}",
-          "permissionsRequired": ["orders-storage.adjustments.item.get"]
-        },
-        {
-          "methods": ["PUT"],
-          "pathPattern": "/orders-storage/adjustments/{id}",
-          "permissionsRequired": ["orders-storage.adjustments.item.put"]
-        },
-        {
-          "methods": ["DELETE"],
-          "pathPattern": "/orders-storage/adjustments/{id}",
-          "permissionsRequired": ["orders-storage.adjustments.item.delete"]
-        }
-      ]
-    },
     {
       "id": "orders-storage.alerts",
-      "version": "3.0",
+      "version": "2.0",
```

Comment:
mistake?

Reply:
"version": "3.0" in the 3.0 release https://github.com/folio-org/mod-orders-storage/blob/v3.0.0/descriptors/ModuleDescriptor-template.json#L38

Reply:
Reverting to 3.0